docs/kubectl-plugins.md

```diff
@@ -14,10 +15,19 @@ kubectl open notebook -n my-namespace my-nb-name
 kubectl open notebook -f notebook.yaml
 ```
 
-Install with:
+### Installation
+
+Release binaries are created for most architectures when the repo is tagged.
+Be aware that moving the binary to your PATH might fail due to permissions
+(observed on mac). If it fails, the script will retry the `mv` with `sudo` and
+prompt you for your password:
 
 ```sh
-# TODO: Install from GitHub release.
-go build ./kubectl/open-notebook && mv open-notebook /usr/local/bin/kubectl-open-notebook
+bash -c "$(curl -fsSL https://raw.githubusercontent.com/substratusai/substratus/main/hack/install_kubectl_plugin.sh)"
```

Comment:
This of course can't be tested fully until we tag a new release. I've been testing it via:

```
wget https://raw.githubusercontent.com/substratusai/substratus/feat/generate-notebook-plugin-release-binaries/hack/install_kubectl_plugin.sh
# change the REPO to point to https://github.com/brandonjbjelland/substratus
# simulate command chain with:
bash -c "$(cat install_kubectl_plugin.sh)"
```